### DISCUSSION TITLE
How does the "Add missing languages" addon work

### DISCUSSION THREAD

**Discussion by EmmanuelBlanchet** [WeblateOrg/weblate#7735] [Q&A]
My current weblate setup is one project with multiple components. I want all components to have the same languages available to translate and thought that the "Add missing languages" addon would be exactly for that.
However, it just doesn't do anything. Not sure what I am doing wrong, as there is nothing to configure for that addon.

Is anyone able to help me out here?

**Comment by nijel** [comment]
Running your own Weblate instance? Are celery workers running?

**Reply by EmmanuelBlanchet** [reply]
I am running a self hosted Weblate instance using docker. I believe celery workers are running as the log is filled with heartbeat tasks such as ``celery-celery stderr | [2022-06-04 10:21:28,848: INFO/MainProcess] Task weblate.utils.tasks.heartbeat[10323994-e98b-4b0e-9959-1dfdced6cb19] received`` (not sure why stderr though?)
If I understand it correctly, receiving webhooks and lazy commits are also handled through celery tasks, both of these things work fine.

**Reply by nijel** [reply]
[This add-on](https://docs.weblate.org/en/latest/admin/addons.html#add-missing-languages) is triggered upon adding language to any of the component and daily to do the sync. You should see `running daily add-on` in the log for the latter.
